backend/src/org/commcare/resources/model/Resource.java

```diff
@@ -75,10 +72,6 @@
     // Resource is in the local environment and ready to install
     public static final int RESOURCE_STATUS_LOCAL = 1;
 
-    // TODO: I think this means the same as "upgade" and should be removed and
-    // replaced by it
-    public static final int RESOURCE_STATUS_PENDING = 2;
```

Comment:
This was pretty much unused: it still had references but nothing setting records to this status.
I've removed the references, and this entry, but I'm not changing the numbering system here because it would require a database migration, which I don't think is worth the effort.